### ISSUE TITLE
aeproject node produces endless loop and doesn't fund wallets

### ISSUE THREAD

**Issue by marc0olo** [aeternity/aepp-aeproject-js#232]
For some reason I am not able to use `aeproject node` anymore to fund wallets. the command starts all containers but that's it.

after waiting for a long time I get this message:
- `Cannot start AE Node!`

**Comment by marc0olo** [comment]
![Bildschirmfoto von 2019-09-14 21-45-54](https://user-images.githubusercontent.com/8124114/64913048-195ace80-d739-11e9-8bb8-0399b36425cb.png)

**Comment by marc0olo** [comment]
so I have 2 problems here:
1. `aeproject init` provides an error (missing .gitignore)
2. `aeprojec node` error cannot start AE Node after waiting ~ 5 minutes

now I switched back to forgAE and got the nodes running and wallets funded

**Comment by Jkat** [comment]
+1 on the `aeproject init` errors too

**Comment by maptuhec** [comment]
Thanks for opening the issue. We will investigate the issues and come up with a fix asap.

**Comment by alekseytsekov** [comment]
Hi guys!
The easy fix for `.gitignore` issue is `cd /usr/local/lib/node_modules/aeproject/aeproject-init/artifacts/ && touch .gitignore`, tested on mac and ubuntu 18.04.
Another issue when the node wont start, happened only once. When try to debug the issue, I write a `console.log` and re-run the command `aeproject node` and the node starts. After that I initialized a new project and try to start the node, it is started from first time.
New release will be available soon. 
![Screenshot from 2019-10-11 15-11-33](https://user-images.githubusercontent.com/15261656/66652734-191df800-ec3f-11e9-8462-1d2f6511c70b.png)
![Screenshot from 2019-10-11 15-05-58](https://user-images.githubusercontent.com/15261656/66652758-26d37d80-ec3f-11e9-820b-6f1149417bf7.png)

**Comment by alekseytsekov** [comment]
Hi @marc0olo,  @Jkat 
if you running on windows and aeproject wont start 
just uninstall current docker and install it again from https://docs.docker.com/docker-for-windows/install
PS: do not click on checkbox for "windows images" or something like this
then node should start

**Comment by marc0olo** [comment]
I updated to 1.0.3 and still have the problem with the funding of wallets when calling `aeproject node`

```
marc0olo@N151ZU:~/git/kryptokrauts/smart-contracts$ mkdir aeternity-new
marc0olo@N151ZU:~/git/kryptokrauts/smart-contracts$ cd aeternity-new/
marc0olo@N151ZU:~/git/kryptokrauts/smart-contracts/aeternity-new$ aeproject init
===== Initializing AEproject =====
===== Installing aepp-sdk =====
===== Installing AEproject locally =====
===== Installing yarn locally =====
===== Creating project file & dir structure =====
===== Creating contracts directory =====
===== Creating tests directory =====
===== Creating integrations directory =====
===== Creating deploy directory =====
===== Creating docker directory =====
==== Adding additional files ====
===== AEproject was successfully initialized! =====
marc0olo@N151ZU:~/git/kryptokrauts/smart-contracts/aeternity-new$ aeproject node
===== Starting node =====
..........................................................................................Cannot start AE Node!
marc0olo@N151ZU:~/git/kryptokrauts/smart-contracts/aeternity-new$ aeproject --version
1.0.3
marc0olo@N151ZU:~/git/kryptokrauts/smart-contracts/aeternity-new$ 
```

I am running it on elementary OS 5.0 Juno (based on Ubuntu 18.04):
```
marc0olo@N151ZU:~$ cat /etc/os-release
NAME="elementary OS"
VERSION="5.0 Juno"
ID=elementary
ID_LIKE=ubuntu
PRETTY_NAME="elementary OS 5.0 Juno"
LOGO=distributor-logo
VERSION_ID="5.0"
HOME_URL="https://elementary.io/"
SUPPORT_URL="https://elementary.io/support"
BUG_REPORT_URL="https://github.com/elementary/appcenter/issues/new"
PRIVACY_POLICY_URL="https://elementary.io/privacy-policy"
VERSION_CODENAME=juno
UBUNTU_CODENAME=bionic
```

**Comment by alekseytsekov** [comment]
Hi @marc0olo 
I install 'elementary' and try to reproduce your issue but I able to run the node with 'sudo docker-compose -f docker-compose.yml' and with 'sudo aeproject node'. 
PS: without 'sudo' execution of 'aeproject node' throws error

![Capture-9](https://user-images.githubusercontent.com/15261656/67207601-9eae5e80-f41c-11e9-89c7-60c2a8f53fa2.PNG)
![Capture-10](https://user-images.githubusercontent.com/15261656/67207625-a79f3000-f41c-11e9-9de9-e5c0a1bba0b8.PNG)
![Capture-11](https://user-images.githubusercontent.com/15261656/67207683-b980d300-f41c-11e9-8dc2-0a0dd33c23de.PNG)
![Capture-12](https://user-images.githubusercontent.com/15261656/67207701-c00f4a80-f41c-11e9-9402-672d5f531c31.PNG)

**Comment by marc0olo** [comment]
mhhh. all I can say is that it doesn't work on my machine. I am wondering why your os-information says `UBUNTU_CODENAME=juno` instead of `bionic`

**Comment by alekseytsekov** [comment]
@marc0olo can you try to run this command in the project root directory
```
sudo docker-compose -f docker-compose.yml -f docker-compose.compiler.yml up
```

**Comment by marc0olo** [comment]
this command works. there are some (probably) strange messages on the nodes:
```
node3_1     | 21:45:13.082 [info] Connection accept failed - aead_chacha20poly1305_ietf_decrypt_failed was from <<"172.18.0.2">>
node2_1     | 21:45:13.979 [info] Connection handshake failed - timeout was from <<"node1">>
node3_1     | 21:45:14.087 [info] Connection accept failed - aead_chacha20poly1305_ietf_decrypt_failed was from <<"172.18.0.3">>
node3_1     | 21:45:14.953 [info] Connection handshake failed - timeout was from <<"node2">>
node3_1     | 21:45:14.958 [info] Connection handshake failed - timeout was from <<"node1">>
node1_1     | 21:45:18.082 [info] Connection handshake failed - timeout was from <<"node3">>
node2_1     | 21:45:18.189 [info] Connection accept failed - aead_chacha20poly1305_ietf_decrypt_failed was from <<"172.18.0.2">>
node2_1     | 21:45:19.087 [info] Connection handshake failed - timeout was from <<"node3">>
node1_1     | 21:45:19.194 [info] Connection accept failed - aead_chacha20poly1305_ietf_decrypt_failed was from <<"172.18.0.3">>
node2_1     | 21:45:19.960 [info] Connection accept failed - aead_chacha20poly1305_ietf_decrypt_failed was from <<"172.18.0.4">>
node1_1     | 21:45:23.190 [info] Connection handshake failed - timeout was from <<"node2">>
node2_1     | 21:45:24.194 [info] Connection handshake failed - timeout was from <<"node1">>
node3_1     | 21:45:24.961 [info] Connection handshake failed - timeout was from <<"node2">>
node1_1     | 21:45:25.069 [info] Connection accept failed - aead_chacha20poly1305_ietf_decrypt_failed was from <<"172.18.0.4">>
node3_1     | 21:45:30.070 [info] Connection handshake failed - timeout was from <<"node1">>
node3_1     | 21:45:33.091 [info] Connection accept failed - aead_chacha20poly1305_ietf_decrypt_failed was from <<"172.18.0.2">>
node3_1     | 21:45:34.096 [info] Connection accept failed - aead_chacha20poly1305_ietf_decrypt_failed was from <<"172.18.0.3">>
node1_1     | 21:45:38.091 [info] Connection handshake failed - timeout was from <<"node3">>
node2_1     | 21:45:38.199 [info] Connection accept failed - aead_chacha20poly1305_ietf_decrypt_failed was from <<"172.18.0.2">>
node2_1     | 21:45:39.096 [info] Connection handshake failed - timeout was from <<"node3">>
node1_1     | 21:45:39.204 [info] Connection accept failed - aead_chacha20poly1305_ietf_decrypt_failed was from <<"172.18.0.3">>
node2_1     | 21:45:39.969 [info] Connection accept failed - aead_chacha20poly1305_ietf_decrypt_failed was from <<"172.18.0.4">>
node1_1     | 21:45:43.200 [info] Connection handshake failed - timeout was from <<"node2">>
node2_1     | 21:45:44.204 [info] Connection handshake failed - timeout was from <<"node1">>
node3_1     | 21:45:44.970 [info] Connection handshake failed - timeout was from <<"node2">>
```

but the nodes are up and running.

**Comment by marc0olo** [comment]
just to explain the issue a bit more -> I don't think the problem is the start of the containers. when calling `docker-compose ps` after running `aeproject node` I see the containers up and running.

it seems like somehow the script cannot detect the running nodes and fund the wallets for some reason.

**Comment by marc0olo** [comment]
```
marc0olo@N151ZU:~/git/aeternity/aepp-aeproject-js/test-ae-node$ sudo aeproject node
[DEBUG] docker image: node1
[DEBUG] wait For Container: [ 'Name   Command   State   Ports',
  '------------------------------',
  '' ]
===== Starting node =====
[DEBUG] wait For Container: [ 'Name   Command   State   Ports',
  '------------------------------',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.[DEBUG] wait For Container: [ '        Name                       Command               State                                       Ports                                     ',
  '-----------------------------------------------------------------------------------------------------------------------------------------------',
  'testaenode_compiler_1   /docker-entrypoint.sh            Up      0.0.0.0:3080->3080/tcp                                                        ',
  'testaenode_node1_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node2_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_node3_1      bin/aeternity console -noi ...   Up      3013/tcp, 3014/tcp, 3015/tcp, 3113/tcp                                        ',
  'testaenode_proxy_1      nginx -g daemon off;             Up      0.0.0.0:3001->3001/tcp, 0.0.0.0:3002->3002/tcp, 0.0.0.0:3003->3003/tcp, 80/tcp',
  '' ]
.Cannot start AE Node!
marc0olo@N151ZU:~/git/aeternity/aepp-aeproject-js/test-ae-node$ 
```

**Comment by marc0olo** [comment]
what seems strange to me is that at an earlier release the docs said that it was just a renaming from **forgae** to **aeproject**. already at this stage I was able to start and fund the nodes with **forgae** but when using **aeproject** it didn't work.

there must have been some change that leads to this strange effect. hope you can find and fix this soon. otherwise it would be good to have a command just to fund the standard wallets. because when running `docker-compose up` manually the stack is up and running. I just don't have the wallets initially funded

**Comment by alekseytsekov** [comment]
Hi @marc0olo 
which version of `docker-compose` do you use? 
```
docker-compose --version
```
if it is lower v1.2, can you try to update it and try again?
```
https://docs.docker.com/compose/install/
```
my current docker-compose version is docker-compose version 1.23.2.
from v1.2 `docker-compose ps` now shows the containers’ health status if available.

**Comment by marc0olo** [comment]
I can check when I am at home but still I don't get why the behaviour changed after renaming from `forgae` to `aeproject`

**Comment by alekseytsekov** [comment]
@marc0olo because earlier we use `docker` and now we use `docker-compose`.
With `docker-compose` we want to support more users and OS, `docker` does not run on windows 7, 8

**Comment by marc0olo** [comment]
just checked the version:
```
marc0olo@N151ZU:~/git/privat$ docker-compose --version
docker-compose version 1.17.1, build unknown
```
will update and tell you whether it works afterwards :+1:

**Comment by marc0olo** [comment]
just updated to docker-compose `1.24.0` and now it works. you should definitely add this to the documentation. thanks!

**Comment by alekseytsekov** [comment]
@marc0olo super! We will add this requirement to the documentation.

**Comment by maptuhec** [comment]
@marc0olo  It is added to the docs since, version 1.0.1. But we should have probably made an announcement that people should upgrade. Thanks for the feedback and sorry for the headaches. 

Best,
Martin

**Comment by maptuhec** [comment]
I will close the issue now, since the issue is fixed
